crypto/rsa/rsa_ossl.c

```diff
@@ -119,7 +119,7 @@ static int rsa_ossl_public_encrypt(int flen, const unsigned char *from,
                                                     from, flen, NULL, 0,
                                                     NULL, NULL);
         break;
-#ifndef FIPS_MODULE
+#if !defined(FIPS_MODULE) && !defined(OPENSSL_NO_DEPRECATED_3_0)
```

Comment:
This change has no effect because you have the `#include "internal/deprecated.h"` above. Unfortunately that means the rsa_ssl.c cannot be among the `!$disabled{'deprecated-3.0'}` files.

Reply:
This is the cause for the no-deprecated build failure.

Reply:
`rsa_ossl` needs to use deprecated stuff and `rsa_ssl` implements deprecated stuff. So it seems that using the standard deprecation flags will not work.  Does anyone have suggestions on how to do this deprecation?  One possibility I am about to try is to add a shim file that doesn't include `internal/deprecated.h` Let's see what happens.

Reply:
No, you're misunderstanding. The fix is just to drop the move of rsa_ssl.c into the `!$disabled{'deprecated-3.0'}` block. Just drop the build.info change.

Reply:
> Just drop the build.info change.

well heck, that's easier. :)